src/FileManager.js

```diff
@@ -0,0 +1,31 @@
+class FileManager {
+  constructor() {
+    this.cohorts = []
+  }
+
+  addCohort(cohort) {
+    this.cohorts.push(cohort)
+  }
+
+  removeCohortByName(name) {
+    const locatedCohort = this.cohorts.find(
+      (cohort) => cohort.CohortName === name
+    )
+    if (!locatedCohort) {
+      throw new Error('This cohort does not exist')
+    }
+    return locatedCohort
+  }
```

Comment:
Yeah, it's as Nazar says. This function name tells us that it should remove a cohort but it doesn't. Do you have a suitable test for this function? If you do, that test should be failing.